opentasks/src/main/java/org/dmfs/tasks/utils/colors/Blended.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2017 dmfs GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.dmfs.tasks.utils.colors;
+
+import org.dmfs.android.bolts.color.Color;
+
+
+/**
+ * Color blended from two others. See implementation for details.
+ *
+ * @author Gabor Keszthelyi
+ */
+public final class Blended implements Color
```

Comment:
Note, the purpose of this code is essentially the reverse of `Mixed`. So if `Mixed` blends the colors `A` and `B` into `C`, this class returns `B` when you give it `A` (`baseColor`), `C` (`targetColor`) and the alpha value of `B`. We use this in the editor to determine the color of the action bar, so it mixes with the background in a way that keeps the result the same (while the alpha of the action bar goes towards opaque)

Reply:
Ok, I've added this note as javadoc to the class.